### PR TITLE
feat: optional question time encouragement

### PR DESCRIPTION
**Pull request by cmeesters**
I want to have optional question times at the end of each content chapter. 

## Release Notes

- **New Features**
  - Added a "Question Time" section to enhance interactivity in presentations, allowing for dedicated Q&A segments based on course configuration.
  - Introduced a "Frustration" section to promote a positive learning environment by addressing common mistakes.

- **Improvements**
  - Enhanced flexibility of presentation content through conditional slide inclusions based on user-defined settings.

- **Style Changes**
  - Modified section headers for better document organization and clarity.

### CODE REVIEW

File: slides/Snakemake_HPC_Users.tex [hunk 1]
```diff
@@ -48,30 +48,38 @@
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \include{common/software_environment}
+{<+++ if course.question is defined +++>\include{common/question_time}e<+++endif+++>
```
[the remark this answers is not in the document]
Comment (cmeesters) [reply slides/Snakemake_HPC_Users.tex]: oh, yes. The mistake has been corrected.

File: slides/Snakemake_HPC_User_Creator_Combi.tex [hunk 2]
```diff
@@ -48,45 +51,56 @@
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \include{common/software_environment}
+{<+++ if course.question is defined +++>\include{common/question_time}e<+++endif+++>
```
[the remark this answers is not in the document]
Comment (cmeesters) [reply slides/Snakemake_HPC_User_Creator_Combi.tex:54]: done

File: slides/Snakemake_HPC_Creators.tex [hunk 3]
```diff
@@ -49,27 +49,35 @@
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \include{common/software_environment}
+{<+++ if course.question is defined +++>\include{common/question_time}e<+++endif+++>
```
[the remark this answers is not in the document]
Comment (cmeesters) [reply slides/Snakemake_HPC_Creators.tex]: done

**Comment by brand-fabian** [comment]
LGTM

**Pull request merged**